elements/bulbs-video/components/revealed.test.js

```diff
@@ -8,6 +8,30 @@ describe('<bulbs-video> <Revealed>', () => {
   describe('propTypes', () => {
     let subject = Revealed.propTypes;
 
+    it('accepts autoplay boolean', () => {
+      expect(subject.autoplay).to.eql(PropTypes.bool);
+    });
+
+    it('accepts autoplayNext boolean', () => {
+      expect(subject.autoplayNext).to.eql(PropTypes.bool);
+    });
+
+    it('accepts noEndcard boolean', () => {
+      expect(subject.noEndcard).to.eql(PropTypes.bool);
+    });
+
+    it('accepts targetCampaignId string', () => {
+      expect(subject.targetCampaignId).to.eql(PropTypes.string);
+    });
+
+    it('accepts targetSpecialCoverage string', () => {
+      expect(subject.targetSpecialCoverage).to.eql(PropTypes.string);
+    });
+
+    it('accepts twitterHandle string', () => {
+      expect(subject.twitterHandle).to.eql(PropTypes.string);
+    });
```

Comment:
Do we need these tests? Seems redundant since there's not really any logic to test here.